ghastly/pebble.py

```diff
@@ -7,7 +7,7 @@ class Pebble:
     and track its material composition and unique id number.
     '''
 
-    def __init__(self, coords, radius, mat_id, pebble_id, recirc=False):
+    def __init__(self, coords, r, reg_id, pass_num, l_type, pebble_id):
```

Comment:
I'm definitely open to changing it - I felt like lammps_type is a bit long, which is why I initially picked l_type.

Reply:
Hmm, if it's the name in LAMMPS, then I think it's ok.

Reply:
In LAMMPS it's just "type", which I can't use in python, since it's a keyword :(

Reply:
Looks like this is the last unresolved comment, so I'll avoid merging until this is resolved. Also, since `l_type` sorts the materials in LAMMPS, maybe `lam_mat_id` or `lammps_id`?